content/doc/developer/internationalization/new-internationalization.adoc

```diff
@@ -0,0 +1,339 @@
+---
+title: New Internationalization
+layout: developersection
+---
+ 
+https://wiki.jenkins.io/display/JENKINS/How+to+view+Jenkins+in+your+language[How
```

Comment:
Broken link.

Reply:
The wiki redirection works now for this one :)